Cargo.toml

```diff
@@ -1,34 +1,19 @@
-[package]
-name = "comprakt"
-version = "0.0.1"
-authors = [
-    "flip1995 <hello@philkrones.com>",
-    "hediet <>",
-    "joshuabach <>",
-    "problame <>",
-    "reiner-dolp <>"
-]
-edition = "2018"
-license = "MIT OR Apache-2.0"
-
 [workspace]
-members = [ ".", "mjtest-rs", "mjtest-rs/mjtest_macros", "asciifile" ]
-
-[dependencies]
-structopt = "0.2"
-failure = "0.1.3"
-memmap = "0.7.0"
-libc = "*"
-termcolor = "1.0.4"
-derive_more = "0.13"
-asciifile = { path = "./asciifile" }
-strum = "0.11.0"
-strum_macros = "0.11.0"
+members = [
+    "compiler-lib",
+    "compiler-cli",
```

Comment:
@flip1995 The only side-effect of listing crates seems to be a shared cargo.lock. Should we also list the integration test crates in compiler-cli?

Reply:
It would at least be more consistent to list them here, since we're also listing  `mjtest-rs/mjtest_macros` here.

Reply:
or remove mjtest from the list